modules/apps/object/object-web/src/main/resources/META-INF/resources/object_definitions/object_definition/relationships.jsp

```diff
@@ -59,9 +59,9 @@ renderResponse.setTitle(objectDefinition.getLabel(locale, true));
 		module="js/components/ObjectRelationship/AddRelationship"
 		props='<%=
 			HashMapBuilder.<String, Object>put(
-				"ffOneToOneRelationshipConfigurationEnabled", objectDefinitionsRelationshipsDisplayContext.isFFOneToOneRelationshipConfigurationEnabled()
-			).put(
 				"objectDefinitionExternalReferenceCode", objectDefinition.getExternalReferenceCode()
+			).put(
+				"objectRelationshipTypes", objectDefinitionsRelationshipsDisplayContext.getObjectRelationshipTypes(objectDefinition)
```

Comment:
Hey @murilostodolni, We have to put this prop in `edit_object_relationship.jsp` too, otherwise we will have a bug that if we edit a created relationship, the `Type` select will not be filled. You can also put the `objectRelationshipTypes` as a mandatory prop in the `ObjectRelationshipFormBaseProps` by removing the question mark